### PR TITLE
Basic Math Parser.js

### PR DESCRIPTION
**Pull request by ynr-ram**
The Basic Math Parser enables users to perform simple addition calculations directly within the chat interface by entering commands like !add 5 10. It captures the two numbers provided in the command, computes their sum, and returns the result to the user in a friendly format. This parser enhances user interaction by allowing quick calculations without the need for external tools or applications.

### CODE REVIEW

File: Parsers/Basic Math Parser.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+activation_example: !add 5 10
+regex: !add (\d+) (\d+)
```
Comment (earlduque) [inline Parsers/Basic Math Parser.js:3]: this regex already conflicts with another one of your pull requests.

File: Parsers/Basic Math Parser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+flags: gmi
+order: 100
+stop_processing: true
+*/
+
+var matches = current.text.match(/!add (\d+) (\d+)/);
+if (matches) {
+    var num1 = parseInt(matches[1], 10);
```
Comment (earlduque) [inline Parsers/Basic Math Parser.js:11]: there are many more ways this can be implemented without being so restrictive on how it works

**Review by earlduque — CHANGES_REQUESTED** [review]
Several of your recent pull requests are getting into the realm of spam, please be more thoughtful about your contributions

**Comment by SapphicFire** [comment]
Thank you for your submission. However, there are outstanding changes required before we can approve this parser. Please consider alternative solutions that make this parser robust and extensible.

Please complete your contribution by adding the necessary elements and resubmit or update this pull request for further review. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

**Pull request closed without merging**